.eslintrc.json

```diff
@@ -13,7 +13,10 @@
   },
   "globals": {
     "__DEV__": true,
-    "IS_DEV": true
+    "IS_DEV": true,
+    "ios": true,
+    "android": true,
+    "akd": true
```

Comment:
any reason we would prefer not to name this `androidKeyboardDismiss` for clarity?

Reply:
hm, i made it like that and didn't revisit. will fix